src/components/hastagLists.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div class="topic" v-for="topic in blog.topics" :key="topic.id" @click="clickedTopic(topic)">
+      <a>  # {{ topic }}</a>
+    </div>
+</template>
+  
+<script setup>
+import { defineProps, defineEmits } from 'vue';
+const props = defineProps({
+    blog: Object,
```

Comment:
hashtag component should only have topics/hashtags not the full data @ShivanshSinghnnovative